src/common/components/About/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Header, AboutText } from '@components';
+
+const About = () => (
+  <React.Fragment>
```

Comment:
Good use of modern React.Fragment piece of framework. Why React.Fragment is a good practice?

Before it had been introduced to React in general, we had to use wrappers, such as div's, sections or any other parent "containers". While it doesn't sound terrible, sometimes it was. Use of div's everywhere is kind of heritage of HTML4 standard, where everything was designed in this way. What sounds bad, useful HTML5 elements such as section or article were often just ignored for some reason.

Now React has improved by use of fragments, so we are not creating redundant blocks. It also had often impact on web accessibility. 

Again, well job, this kind of pattern that you have used complies with modern, good HTML & JS standards.